### PR TITLE
Hometask #7: Routing

### PR DESCRIPTION
**Pull request by MarharytaKrasikava**
Task description: https://epam-my.sharepoint.com/personal/andrey_pavlov_epam_com/_layouts/15/onedrive.aspx?id=/personal/andrey_pavlov_epam_com/Documents/AngularMentoring/7_Routing.pdf&parent=/personal/andrey_pavlov_epam_com/Documents/AngularMentoring&originalPath=aHR0cHM6Ly9lcGFtLW15LnNoYXJlcG9pbnQuY29tLzpiOi9wL2FuZHJleV9wYXZsb3YvRWVZcHVXdGZSNkZLaklCRmgybWFwRmdCSUw1MHVWMm5LWXUybjJvazZaU3Fmdz9ydGltZT10ZWdsX20tUjJFZw

### CODE REVIEW

File: src/app/video-courses/new-course/new-course.component.ts [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+  public showBreadCrumbs: boolean = false;
+
+  constructor(private route: ActivatedRoute, private courseService: VideoCoursesService, private router: Router) {}
+
+  public ngOnInit(): void {
+    const id: string = this.route.snapshot.params.id;
+    this.course = this.courseService.getCourse(id);
+    if (this.course) {
+      this.showBreadCrumbs = true;
+    }
```
Comment (Yauhenia-Zhdanovich) [inline src/app/video-courses/new-course/new-course.component.ts]: `this.showBreadCrumbs = !!this.course;`

File: src/app/video-courses/new-course/new-course.component.ts [hunk 2]
```diff
@@ -1,17 +1,33 @@
...
 
 @Component({
   selector: 'app-new-course',
   templateUrl: './new-course.component.html',
   styleUrls: ['./new-course.component.css']
 })
-export class NewCourseComponent {
+export class NewCourseComponent implements OnInit {
+  public course: Course;
+  public showBreadCrumbs: boolean = false;
```
Comment (Yauhenia-Zhdanovich) [inline src/app/video-courses/new-course/new-course.component.ts]: why would you need this property?

File: src/app/video-courses/page-not-found/page-not-found.component.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-page-not-found',
+  templateUrl: './page-not-found.component.html',
+  styleUrls: ['./page-not-found.component.css']
+})
+export class PageNotFoundComponent {
+
+  // constructor(private router: Router) { }
```
Comment (Yauhenia-Zhdanovich) [inline src/app/video-courses/page-not-found/page-not-found.component.ts]: pls delete unnecessary comments

File: src/app/video-courses/new-course/new-course.component.ts (hunk 1 above)
Comment (MarharytaKrasikava) [reply src/app/video-courses/new-course/new-course.component.ts]: Removed

File: src/app/video-courses/page-not-found/page-not-found.component.ts (hunk 3 above)
Comment (MarharytaKrasikava) [reply src/app/video-courses/page-not-found/page-not-found.component.ts]: Done

**Pull request merged**